src/alto-ui/Tabs/__tests__/Tabs.test.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Tabs from '../Tabs';
+
+describe('Tabs', () => {
+
+    beforeEach(() => {
+        jest.clearAllMocks()
+    });
+
+    const getWrapper = (defaultProps, props) => mount(<Tabs {...defaultProps} {...props} />)
```

Comment:
`defaultProps` should be always on top of the describe and this function should looks like:
```
const defaultProps = { ... };
const getWrapper = props => mount(<Tabs {...defaultProps} {...props} />);
```

and if you want to change something in `defaultProps` you should use it like:
```
describe('something', () => {
  let currentDefaultProps;
  beforeAll(() => {
    currentDefaultProps = { ...defaultProps, someProp: 'abc' };
  });
  it('is some test', () => {
    const wrapper = getWrapper({ someOtherProp: 'yoyo' });
  });
});
```